ts/src/okx.ts

```diff
@@ -1792,7 +1792,9 @@ export default class okx extends Exchange {
             if (limit !== undefined) {
                 request['limit'] = limit; // default 100
             }
-            response = await this.publicGetMarketTrades (this.extend (request, params));
+            let method = undefined;
+            [ method, params ] = this.handleOptionAndParams (params, 'fetchTrades', 'method', 'publicGetMarketTrades');
+            response = await this[method] (this.extend (request, params));
```

Comment:
@ttodua let's avoid dynamic calls (`this[method`]), we can easily do 
```
if (method === 'publicGetMarketTrades') {
   response  = await this.publicGetMarketTrades()
} else {
 ....
}
```

Reply:
uh oh, i was about to fix soon. please disregard the PR's were you had left notes and I've not yet addressed them.